flask_unchained/bundles/oauth/views/oauth_controller.py

```diff
@@ -0,0 +1,34 @@
+from flask_unchained import Controller, current_app, request, route, session, url_for, injectable
+
+from ..extensions import OAuth
+
+
+class OAuthController(Controller):
+    oauth: OAuth = injectable
+
+    @route(methods=['GET', 'POST'])
```

Comment:
All of these routes can (should) be `GET` requests (`GET` is the default, so just `@route()` is sufficient)